_projects/2019/sunpy/helioviewer.md

```diff
@@ -24,25 +24,20 @@ collaborating_projects:
 
 #### Description
 
-This project aims to design and implement a low level Python wrapper around the
-unirest API provided by the HelioViewer Project. This would create a new affiliated SunPy package (name up for suggestions).
+This project aims to design and implement a low level Python wrapper around the unirest API provided by the HelioViewer Project. This would create a new affiliated SunPy package (name up for suggestions).
 
 Currently SunPy has a `HelioviewerClient` that implements only a handful of the available API.
-They are `getClosestImage`, `getJP2Image` and `takeScreenshot` from the Helioviewer API and you can see our code [here.](https://github.com/sunpy/sunpy/blob/master/sunpy/net/helioviewer.py)
+They are `getClosestImage`, `getJP2Image`, `getJP2Header (in an open pull request as of writing)` and `takeScreenshot` from the Helioviewer API and you can see our code [here.](https://github.com/sunpy/sunpy/blob/master/sunpy/net/helioviewer.py)
 
-What we want to do is to create a new Python package that will lightly wrap every aspect of the HelioViewer API allowing anyone to create their own helioviewer.org like GUI in Python (this could be part of the project but will depend on the progress).
-
-To facilitate this, we will need to remove the current code from SunPy and move it to the new affiliated package.
-#### Secondary goals
-
-* Start on a GUI-like interface for the new package, if time permits.
+What we want to do is to create a new Python package that will lightly wrap every aspect of the HelioViewer API allowing anyone to create their own helioviewer.org like GUI in Python (this could be part of the project but will depend on the progress). To facilitate this, we will need to remove the current code from SunPy and move it to the new affiliated package.
 
 #### Milestones
 
-##### Coding starts
+##### Coding starts (time table might be too slow)
```

Comment:
I'm not sure about this note. What does it suggest for students browsing the project ideas?

Reply:
I'll remove it too.